autoload/tagbar.vim

```diff
@@ -2866,6 +2866,46 @@ function! s:EscapeCtagsCmd(ctags_bin, args, ...) abort
     return ctags_cmd
 endfunction
 
+" python simulate system() on window to prevent temporary window creation
+function! s:system_python(cmd, version) abort
```

Comment:
Lets rename this function to `s:run_system()` or something like that since it may or may not actually execute Python at all.

Reply:
done